subprojects/build-cache/src/test/groovy/org/gradle/caching/internal/controller/NextGenBuildCacheControllerTest.groovy

```diff
@@ -98,7 +98,7 @@ class NextGenBuildCacheControllerTest extends Specification {
         def actualSnapshot = controller.createSnapshot(TreeType.FILE, root, manifestEntries)
 
         then:
-        actualSnapshot.empty
+        !actualSnapshot.present
```

Comment:
❓ Why did this need to change?

Reply:
This test fails on Java8, because Optional.empty() is Java11 API :/